documentation/modules/creating-migration-quick-plan-existing-source.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="creating-migration-quick-plan-existing-source_{context}"]
+= Creating and running a migration plan starting on Plans for virtualization
+
+.Procedure
+
+. In the {ocp} web console, click *Plans for virtualization* and then click *Create Plan*.
++
+The  *Create migration plan* wizard opens to the *Select source provider* interface.
+. Select the source provider of the VMs you want to migrate.
++
+The *Select virtual machines* interface opens.
+. Select the VMs you want to migrate and click *Next*.
++
+The *Create migration plan* pane opens. It displays the source provider's name and suggestions for a plan name, a target provider and namespace, a network map, and a storage map.
```

Comment:
`It displays the source provider's name and suggestions for a plan name` - this has changed, now this field (of the plan name) is empty when the wizard is shown

Reply:
@ahadas That's a good change. Adding "enter a name" as a step.